Makefile

```diff
@@ -35,6 +35,9 @@ test: ## Run unit tests
 test: vendor/bin/phpunit
 	$< $(PHPUNIT_OPTS)
 
+sa: ## Run Static Analysis
```

Comment:
fun fact - I always saw it as Static Code Analysis. It took me a while to figure out what `sa` shortcut stands for

Reply:
Yeah, but you don't do Code Unit Tests, right 😝? Personally I always use just "static analysis" term, code is implicit there. But yeah, it's subjective 🙂